packages/frontend-2/lib/projects/helpers/types.ts

```diff
@@ -0,0 +1,13 @@
+import { Get } from 'type-fest'
+import { ProjectWebhooksQuery } from '~~/lib/common/generated/gql/graphql'
+
+export type WebhookItem = NonNullable<
+  Get<ProjectWebhooksQuery, 'project.webhooks.items[0]'>
+>
+
+export type FormValues = {
```

Comment:
Way too vague of a name considering that this is defined in a `projects` helper (there are so many different forms relating to projects)

Reply:
These naming rules might seem unimportant, but they do in fact meaningfully improve readability & maintainability of the project, especially when there's multiple people working on it.

If you ever have the time I suggest checking out a classic programming book called "Clean Code" (at least the 1st part of the book). It's oriented around Java, but the same rules apply to all programming generally. 

Dunno how legal it is, but I found this PDF on the 1st page of Google lol https://thixalongmy.haugiang.gov.vn/media/1175/clean_code.pdf